pkg/apis/crds/karpenter.sh_provisioners.yaml

```diff
@@ -6,6 +6,10 @@ metadata:
     controller-gen.kubebuilder.io/version: v0.8.0
   creationTimestamp: null
   name: provisioners.karpenter.sh
+  labels:
```

Comment:
Should we be adding this? These labels are generally automatically injected by Helm when the chart is first installed?